src/Resources/Resources/help/Save-AzDeploymentTemplate.md

```diff
@@ -25,20 +47,20 @@ Save-AzDeploymentTemplate -DeploymentObject <PSDeployment> [-Path <String>] [-Fo
 ```
 
 ## DESCRIPTION
-The **Save-AzDeploymentTemplate**  cmdlet saves a deployment template to a JSON file.
+The **Save-AzDeploymentTemplate** cmdlet saves a deployment template to a JSON file.
 
 ## EXAMPLES
 
-### Example 1
+### Example 1 Save template for a deployment at subscription scope
 ```powershell
-PS C:\> Save-AzDeploymentTemplate -DeploymentName "TestDeployment"
+PS C:\> Save-AzDeploymentTemplate -ScopeType "Subscription" -DeploymentName "TestDeployment"
```

Comment:
@Tiano2017 same comment about adding the corresponding output to all examples in this file (if this cmdlet does return anything)